src/protocols/two_party_ecdsa/lindell_2017/test.rs

```diff
@@ -87,5 +87,27 @@ mod tests {
             &encrypted_pairs,
             &proof,
         ).expect("range proof error");
+
+        // pdl proof minus range proof
```

Comment:
Is this sufficient testing? Are this the only scenario you can think about?

Reply:
it is not a scenario actually. PDL proof includes also the range proof but they can be seperated which is what we are doing here. The protocol is self testing - both parties have a strong verification step in this protocol.